shopfloor_printing_base/components/printing.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.addons.component.components.base import AbstractComponent
+
+
+class ShopFloorPrintingAction(AbstractComponent):
+    """Base Component for actions"""
+
+    _name = "shopfloor.printing.action"
+    _inherit = "shopfloor.process.action"
+    _collection = "shopfloor.printing"
+
+    @property
+    def report_to_print(self):
+        """
+        Returns the report to print
+        """
+        return self.work.menu.label_print_report_id
```

Comment:
IMO instead of relying on a fixed report per menu item, we should pass this id into the menu data for the frontend and then the frontend will pas the id of the report to print. This way, the endpoint will work with any report.
Then I don't think we need this endpoint at all since we could use the standard endpoint for printing reports or a similar one.
Just ensure that you can pass the config to the menu or to the scenario.